### ISSUE TITLE
[EpicLoot] [Suggestion] - Giving useless/excess items a use

### ISSUE THREAD

**Issue by NeonCarbide** [RandyKnapp/ValheimMods#119]
I was just scrolling through the recipes.json as I realized a couple things: 

- Amber and Amber Pearls so far only have one use: to sell for Coins
- A number of the material upgrade recipes use Coins as a catalyst

Perhaps, instead of having to go to Haldor to exchange the amber items for Coins, some of the recipes for upgrading materials could incorporate amber items directly.

A couple examples I had mind were the Runestones and the Shards

Currently, the recipes are:

Input | Output | \| | Input | Output
--- | --- | --- | --- | ---
Magic Runestone x5 | Rare Runestone x1 | \| | Magic Shard x5 | Rare Shard x1
Rare Runestone x5<br>Coins x5 | Epic Runestone x1 | \| | Rare Shard x5<br>Coins x5 | Epic Shard x1
Epic Runestone x5<br>Coins x10 | Legendary Runestone x1 | \| | Epic Shard x5<br>Coins x10 | Legendary Shard x1

My proposed recipes for including amber items would be:

Input | Output | \| | Input | Output
--- | --- | --- | --- | ---
Magic Runestone x3<br>Amber Pearl x1 | Rare Runestone x1 | \| | Magic Shard x3<br>Amber x1 | Rare Shard x1
Rare Runestone x3<br>Amber Pearl x1 | Epic Runestone x1 | \| | Rare Shard x3<br>Amber x1 | Epic Shard x1
Epic Runestone x3<br>Amber Pearl x1 | Legendary Runestone x1 | \| | Epic Shard x3<br>Amber x1 | Legendary Shard x1

**Comment by RandyKnapp** [comment]
Hey! That's a good idea. Feel free to set that up in the recipe config and tell me how it goes

**Comment by NeonCarbide** [comment]
I've set it up in my recipe config, along with quite [a few other changes](https://gist.github.com/NeonCarbide/66642d63597b0ef09e120ec7622708be), gonna give it a test in [my modpack](https://valheim.thunderstore.io/package/NeonCarbide/NeonPack/) to see how balanced it is.

I tried to balance secondary ingredient costs so that converting the enchanting material from shards is less efficient than upgrading the material's rarity. I did it this way to give incentive to go hunting for enchanted loot to sacrifice rather than just trophies. I should also mention that I've configured both `Epic Loot` and `Creature Level and Loot Control` to limit trophy drops to only one per creature, because, logically, a creature only has one head (unless someone puts in a hydra...).
